.github/workflows/image-optimization.yml

```
@@ -1,22 +1,22 @@
-    name: calibreapp/image-actions
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout Repo
-        uses: actions/checkout@v3
-
-      - name: Compress Images
-        uses: calibreapp/image-actions@main
-        with:
-          githubToken: ${{ secrets.GITHUB_TOKEN }}
+name: Image optimization
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)